ui/src/components/empty-state/empty-state.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button } from 'design-system/components/button/button'
+import { STRING, translate } from 'utils/language'
+import { useFilters } from 'utils/useFilters'
+import styles from './empty-state.module.scss'
+
+export const EmptyState = () => {
+  const { filters, isActive: filtersActive, clearFilter } = useFilters()
+
+  return (
+    <div className={styles.wrapper}>
+      <span>
+        {translate(
+          filtersActive
+            ? STRING.MESSAGE_NO_RESULTS_FOR_FILTERING
+            : STRING.MESSAGE_NO_RESULTS
+        )}
+      </span>
+      {filtersActive && (
```

Comment:
If we have no active filters, but still no items to show, we might want to suggest creating an new entity and show a button? Kept it simple for now with just the text.